spec/facility_spec.rb

```diff
@@ -2,25 +2,266 @@
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+      #require the facility to have the service you're performing
+    end
+  end
+
+  #decided not to make a test for my helper method `set_plate_type` 
+  #because those tests are relevant and included in my other tests
+
+  describe '#F1 registered vehicles' do
+    it 'checks registered vehicles in facility_1' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.registered_vehicles).to eq([@cruz])
+      #require the facility to have the service you're performing
+    end
+  end
+
+  describe '#F1 collected fees' do
+    it 'checks collected fees in facility_1' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq(100)
+    end
+  end
+
+  describe '#register camaro and check info' do
+    it 'registers camaro and checks info' do
+      expect(@camaro).to be_an_instance_of(Vehicle)
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@camaro)
+      expect(@camaro.registration_date).to eq(Date.today)
+      expect(@camaro.plate_type).to eq(:antique)
+      #require the facility to have the service you're performing
+    end
+  end
+
+  describe '#register bolt and check info' do
+    it 'registers bolt and checks info' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@bolt).to be_an_instance_of(Vehicle)
+      @facility_1.register_vehicle(@bolt)
+      expect(@bolt.registration_date).to eq(Date.today)
+      expect(@bolt.plate_type).to eq(:ev)
+      #require the facility to have the service you're performing
+    end
+  end
+
+  describe '#check F1 registered vehicles and fees again' do
+    it 'checks the registered vehicles after all three are added' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@camaro)
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro, @bolt])
+      expect(@facility_1.collected_fees).to eq(325)
+    end
+  end
+
+  describe '#check facility_2(F2) details' do
+    it 'checks the vehicles, services and fees of facility_2' do
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.services).to eq([])
+    end
+  end
+
+  describe '#add bolt and check F2 details again' do 
+    it 'registers bolt and checks F2 details again' do 
+      @facility_2.register_vehicle(@bolt)
+      expect(@facility_2.register_vehicle(@bolt)).to eq(nil)
+      expect(@facility_2.registered_vehicles).to eq([])
+      expect(@facility_2.collected_fees).to eq(0)
+      # expect(facility_2.registered_vehicles).to eq([])
+      # expect(facility_2.collected_fees).to eq(0)
+    end
+  end
+#written test
+  describe '#written test' do
```

Comment:
Great use of `describe` to organize groups of test examples. Nice work covering the various scenarios for written/license/renew, as well.